src/scripts/templates/unit_activity.html

```diff
@@ -1,5 +1,14 @@
 <div class="panel panel-default">
     <div class="panel-heading">
+        <h2>Profile</h2>
```

Comment:
FYI, the advantage of keeping these two functions separate is that we could reuse the views elsewhere. For instance, if we wanted a dedicated "activity" page instead of putting it inside a unit, we could reuse this view. If we add in this profile bit, we wouldn't be able to. But since it's been so long since this has been removed, we can just include it for now and come back to it later if we want to separate them.